application/__init__.py

```diff
@@ -46,6 +47,24 @@
 else:
     app = Flask(__name__, static_folder="../client/build")
 
+
+@app.errorhandler(Exception)
+def exception_handler(error):
+    if isinstance(error, HTTPException):
+        app.logger.exception(str(error))
+        return error.get_response(), error.code
+
+    app.logger.exception(error)
+    return jsonify(error=str(error)), 500
+
+
+@app.after_request
+def handle_after_request(response: Flask.response_class):
+    if response.content_type == 'application/json':
```

Comment:
Some routes that return json as a response do not necessarily mark the content_type as "application/json". e.g., in route /sftp/ls", https://github.com/junhaoliao/iCtrl/blob/441dad7621489b3549b986289f90fb6987d57a0c/application/routes/sftp.py#L51

We shall replace all `json.dumps()` with `jsonify()` from the Flask library.

Reply:
Let's postpone that to another PR. We will merge this PR first to unblock others.